client/src/components/ChangesModal.js

```diff
@@ -0,0 +1,75 @@
+import React, {useState} from 'react'
+import { 
+    Modal,
+    Box,
+    Typography,
+    Grid
+ } from '@mui/material'
+ import { IoWarningOutline } from "react-icons/io5";
+ import { styled } from '@mui/material/styles';
+ import { StyledButton } from './ProjectForm';
+ import { Link } from 'react-router-dom';
+
+
+ const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 400,
+  bgcolor: 'background.paper',
+  border: '2px solid #000',
+  boxShadow: 24,
+  p: 4,
+};
+
+
+ export default function ChangesModal({open, onClose, handleClose}) {
+    return (
+      
+      <Modal
+        open={open}
+        onClose={onClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description"
+      >
+        <Box sx={style}>
+          <Typography sx={{textAlign: 'center'}} id="modal-modal-title" variant="h4" component="h3">
+            <IoWarningOutline color="red" />
+         </Typography>
+
+          <Typography sx={{textAlign: 'center'}} id="modal-modal-title" variant="h4" component="h4">
+           Wait! You made some changes.
+          </Typography>
+          <Typography id="modal-modal-description" sx={{ mt: 2 }}>
```

Comment:
Please center this text

Reply:
@Spiteless  fixed

Reply:
@Spiteless  All changes are made and ready for review

Reply:
There's an error somewhere with the icon imports:

```bash
[0] Mongoose connected from runServer() and is listening on 4000
[1] Browserslist: caniuse-lite is outdated. Please run:
[1]   npx update-browserslist-db@latest
[1]   Why you should do it regularly: https://github.com/browserslist/update-db#readme
[1] Failed to compile.
[1] 
[1] ./src/components/ChangesModal.js
[1] Module not found: Can't resolve '@mui/icons-material/WarningAmberRounded' in '/Users/trilliumsmith/code/VRMS/VRMS3/client/src/components'
[1] Compiling...
[1] Failed to compile.
[1] 
[1] ./src/components/ChangesModal.jsx
[1] Module not found: Can't resolve '@mui/icons-material/WarningAmberRounded' in '/Users/trilliumsmith/code/VRMS/VRMS3/client/src/components'
```

Not sure what's causing that, it was happening too with the react icon you used earlier and I just added it to the `package.json` to test.

This is on top of navigating to `./client/` and running `yarn install` again